src/services/UtilService.ts

```diff
@@ -0,0 +1,87 @@
+import { api, hasError } from '@/adapter';
+import logger from '@/logger';
+
+const fetchShipmentMethodTypeDesc = async (query: any): Promise <any>  => {
+  return api({
+    url: "performFind",
+    method: "get",
+    params: query
+  });
+}
+
+const fetchStatusDesc = async (query: any): Promise <any>  => {
+  return api({
+    url: "performFind",
+    method: "get",
+    params: query
+  });
+}
+
+const fetchStoreCarrierAndMethods = async (query: any): Promise <any>  => {
+  return api({
+    url: "performFind",
+    method: "get",
+    params: query
+  });
+}
+
+const fetchFacilityContactDetails = async (facilityIds: Array<string>): Promise<any> => {
```

Comment:
This service is not used anywhere and similar service is defined in OrderService file